cmd/node/config/config.toml

```diff
@@ -805,6 +805,9 @@
     StableTagLocation = "https://api.github.com/repos/multiversx/mx-chain-go/releases/latest"
     PollingIntervalInMinutes = 65
 
+[GatewayMetricsConfig]
+    URL = "https://testnet-gateway.multiversx.com"
```

Comment:
I would leave this blank and add a TODO to change this on testnet/devnet/mainnet
In this way it won't show misleading info whenever we start a local testnet